util/data_stream.go

```diff
@@ -97,19 +111,19 @@ type fieldEntry struct {
 	aType string
 }
 
-func NewDataStream(basePath string, p *Package) (*DataStream, error) {
+func NewDataStream(basePath string, p *Package) (*DataStream, *BaseDataStream, error) {
```

Comment:
The `util` package contains structs and methods which can be used in other repositories. I woudn't change signatures of exported methods.